.github/workflows/integration_tests.yml

```diff
@@ -351,10 +351,10 @@ jobs:
         if: matrix.target_platform == 'Desktop' && !cancelled()
         run: |
           python scripts/gha/desktop_tester.py --testapp_dir ta
-      - name: Run iOS integration tests on Simulator locally
-        if: contains(env.mobileTestOn, 'simulator') && matrix.target_platform == 'iOS' && !cancelled()
+      - name: Run iOS integration tests on iOS Simulator / Android Emulator locally
+        if: contains(env.mobileTestOn, 'simulator') && startsWith(matrix.os, 'macos') && matrix.target_platform != 'Desktop' && !cancelled()
```

Comment:
Instead of matrix.target_platform != Desktop, could we explicitly check for matrix.target_platform == ios or android?

Reply:
I can make changes, but my consideration is:
1) this step will be deleted soon. Because I'm gonna use two test jobs/matrixes (one for iOS, one for Android) instead
2) currently. simulator test should be disabled.